src/PerfView/EventViewer/EventWindow.xaml.cs

```diff
@@ -1286,6 +1300,11 @@ public void Update()
                 StatusBar.LogError(fqepEx.Message);
                 m_source.FilterQueryExpressionTree = null; 
             }
+            catch(Exception ex)
+            {
+                StatusBar.LogError(ex.Message);
```

Comment:
I don't think this quite did what I wanted - the error seems to be overwritten by the number of records found, but it does at least write something to the log, so that's good.  I think this will be sufficient for now, and we can see if we need to something here in the future.